AITS/TemperatureAIT.C

```diff
@@ -0,0 +1,82 @@
+#include "TemperatureAIT.H"
```

Comment:
Together with the comment about the name `AIT`, this file does not need to have a different name.

It could either be `Temperature` in a namespace that implies volume coupling, or something like `TemperatureVolume`.